pom.xml

```diff
@@ -64,13 +64,11 @@
         <findbugs.threshold>Low</findbugs.threshold>
     </properties>
     <name>${extension.name} API</name>
-    <description>Project GlassFish Enterprise JavaBean API</description>
+    <description>Java APIs for XML based RPC</description>
     
-    <url>http://metro.java.net/</url>
-
     <organization>
-        <name>GlassFish Community</name>
-        <url>https://glassfish.java.net</url>
+        <name>Oracle Corporation</name>
+        <url>http://www.oracle.com</url>
     </organization>
     <licenses>
         <license>
```

Comment:
License URL should be https://oss.oracle.com/licenses/CDDL+GPL-1.1

Reply:
Done the change.